chain-tx-enclave/enclave-ra/ra-client/src/verifier.rs

```diff
@@ -51,7 +51,7 @@ impl EnclaveCertVerifier {
     }
 
     /// Verifies certificate
-    fn verify_cert(&self, certificate: &[u8]) -> Result<(), EnclaveCertVerifierError> {
+    pub fn verify_cert(&self, certificate: &[u8]) -> Result<Vec<u8>, EnclaveCertVerifierError> {
```

Comment:
Can you update the documentation specifying what this function returns?

Reply:
Done